packages/action-group/stories/action-group.stories.ts

```diff
@@ -89,9 +165,7 @@ export const iconsOnly = (): TemplateResult => {
                 </sp-icon>
             </sp-action-button>
             <sp-action-button label="Info">
-                <sp-icon slot="icon">
-                    ${InfoIcon({ hidden: true })}
-                </sp-icon>
+                <sp-icon slot="icon">${InfoIcon({ hidden: true })}</sp-icon>
```

Comment:
Correct me if I'm wrong, but I thought we weren't using these types of icons anymore. Should this be changed to the other format?